Implemented the Fourier Normalizing Radial Gradient Filter (#17)
sunpy/sunkit-image: changelog/17.feature.rst

> Added a new function (`sunkit_image.offlimb.fnrgf`) to normalize the radial brightness gradient using a fourier approximation.

Reviewer: ```suggestion
Added a new function (`sunkit_image.offlimb.fnrgf`) to normalize the radial brightness gradient using a Fourier approximation.
```

> Added a new function (`fourier_normalizing_radial_gradient_filter`) to normalize the radial brightness gradient using a fourier approximation.

Reviewer: ```suggestion
Added a new function (`sunkit_image.offlimb_enhance.fourier_normalizing_radial_gradient_filter`) to normalize the radial brightness gradient using a fourier approximation.
```

These imports are getting way too long.

Author: Should I change the function name to FNRGF?? It will shorten the imports

Reviewer: Sure, try that and see how it goes. I also think the file itself needs renaming. I am not a fan of offlimb_enchance.

Author: Yes me too. I feels a little odd.

Author: Why not only ```offlimb```?? For the algorithms affecting the surface we can use the file name `surface`

Reviewer: Well what about ones that affect both the surface and the limb?

Reviewer: We could call it enhance or contrast?

Author: Enhance would be much better for algorithms affecting both.

Reviewer: That would work.

> Added a new function (`fourier_normalizing_radial_gradient_filter`) to normalize the radial brightness gradient using a fourier approximation.

Reviewer: ```suggestion
Added a new function (`sunkit_image.enhance.fnrgf`)  to normalize the radial brightness gradient using a fourier approximation.
```

> Added a new feature ```fourier_normalizing_radial_gradient_filter``` to normalize the radial brightness gradient using a fourier approximation.

Reviewer: `fourier_normalizing_radial_gradient_filter` should be a full sphinx interlink.

Author: I hope it is fixed.
